test/SlowTests/Sharding/Subscriptions/ShardedSubscriptionSlowTests.cs

```diff
@@ -355,7 +354,7 @@ public async Task CanUpdateSubscriptionToStartFromDoNotChange()
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Cannot set CV by admin in sharded subscription")]
```

Comment:
let's open an issue for that

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-18223